environment/front.tf

```diff
@@ -111,7 +85,17 @@ data "aws_iam_policy_document" "front_query_secretsmanager" {
     actions = [
       "secretsmanager:GetSecretValue"
     ]
-    resources = ["*"]
+    resources = [
```

Comment:
I think this is still a bit relaxed but didn't want to spend too long working out exactly what the app code needs. Any advice welcomed :)